### ISSUE TITLE
docs publishing should not be nested in subdirectories

### ISSUE THREAD

**Issue by benbrackenc74** [Cycling74/xebra.js#18]
The docs are currently published in a directory structure like this:

xebra.js/[version_number]/

In order for GitHub Pages to work correctly, the index.html needs to be in the root directory on the gh-pages branch.

Maybe just fix this up with some postpublish script.

**Comment by benbrackenc74** [comment]
Maybe this has been fixed with @fde31 's new jaguarjs-docs template? Will test with the next release.

**Comment by fde31** [comment]
I fixed this in my fork of the jaguarjs-jsdoc repo, which can be found here https://github.com/fde31/jaguarjs-jsdoc/tree/xebra_docs

And I remember I published the docs once as a test (to see the correct version number) and it was okay as in 'without subdirs'. So I'm assuming you had an old version of the package around that still generated the subdir structure. It's a bit annoying tbh and I think the way to solve this is by running a `npm update`. Basically this will check for updated deps and according to https://github.com/npm/npm/issues/1727 Git URLs are always updated, which I assume is exactly what we want in that case. Might wanna consider making this part of the `preversion` script?
